apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UpdateObjectOptions.cs

```diff
@@ -125,7 +130,6 @@ internal void ModifyRequest(UpdateRequest request, Object obj)
                 if (IfMetagenerationMatch != null)
                 {
                     request.IfMetagenerationMatch = IfMetagenerationMatch;
-                    RetryHandler.MarkAsRetriable(request);
```

Comment:
We have removed the `RetryHandler.MarkAsRetriable` but haven't added it back later.